src/helpers/permissionHelpers.ts

```diff
@@ -13,9 +30,28 @@ export function getRequiredPermissions(): bigint[] {
 }
 
 export function memberIsModerator(member: GuildMember): boolean {
-	return member.permissions.has(Permissions.FLAGS.KICK_MEMBERS);
+	return member.permissions.has(Permissions.FLAGS.KICK_MEMBERS, true);
```

Comment:
[The `checkAdmin` parameter](https://discord.js.org/#/docs/discord.js/stable/class/Permissions?scrollTo=has) is `true` by default; specifying it here is redundant.

```suggestion
	return member.permissions.has(Permissions.FLAGS.KICK_MEMBERS);
```